frontend/app/routes/user/projects.js

```diff
@@ -11,6 +11,13 @@ export default Route.extend({
     });
   },
 
+  afterModel(model) {
+    // if a user has NOT created any projects, reroute them to the intro page
```

Comment:
The logic here says something more along the lines of:

> if no new projects are found, reroute to the intro page

Because I haven't really work on this code as much as you, i don't know where project _creation_ happens, I don't have that context.